tests/e2e/support/api/external/openWithWeb.ts

```diff
@@ -0,0 +1,22 @@
+import join from 'join-path'
+import { checkResponseStatus, request } from '../http'
+import { User } from '../../types'
+
+export const getUrl = async ({
```

Comment:
this should prevent the name clash in the future
```suggestion
export const getOpenWithWebUrl = async ({
```